<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dvbcutbase</class>
 <widget class="QMainWindow" name="dvbcutbase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dvbcut</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QListView" name="eventlist">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="imagedisplay">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="goinput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gobutton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>go</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="picinfolabel">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="goinput2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gobutton2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>go</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="pictimelabel">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="picinfolabel2">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="pictimelabel2">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="linslider">
      <property name="maximum">
       <number>180000</number>
      </property>
      <property name="singleStep">
       <number>7500</number>
      </property>
      <property name="pageStep">
       <number>45000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>90000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="jogslider">
      <property name="minimum">
       <number>-100000</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="singleStep">
       <number>10000</number>
      </property>
      <property name="pageStep">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>100000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="recentfilespopup">
     <property name="title">
      <string>Open recent...</string>
     </property>
    </widget>
    <addaction name="fileNewAction"/>
    <addaction name="fileOpenAction"/>
    <addaction name="recentfilespopup"/>
    <addaction name="fileSaveAction"/>
    <addaction name="fileSaveAsAction"/>
    <addaction name="snapshotSaveAction"/>
    <addaction name="chapterSnapshotsSaveAction"/>
    <addaction name="separator"/>
    <addaction name="fileExportAction"/>
    <addaction name="separator"/>
    <addaction name="fileCloseAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="editconvertpopup">
     <property name="title">
      <string>Convert bookmarks</string>
     </property>
     <addaction name="editConvertStart"/>
     <addaction name="editConvertStop"/>
     <addaction name="editConvert4_3"/>
     <addaction name="editConvert16_9"/>
    </widget>
    <addaction name="editStartAction"/>
    <addaction name="editStopAction"/>
    <addaction name="editChapterAction"/>
    <addaction name="editBookmarkAction"/>
    <addaction name="separator"/>
    <addaction name="editAutoChaptersAction"/>
    <addaction name="editSuggestAction"/>
    <addaction name="editImportAction"/>
    <addaction name="editconvertpopup"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="viewNormalAction"/>
    <addaction name="viewUnscaledAction"/>
    <addaction name="viewDifferenceAction"/>
    <addaction name="separator"/>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="separator"/>
    <addaction name="viewFullSizeAction"/>
    <addaction name="viewHalfSizeAction"/>
    <addaction name="viewQuarterSizeAction"/>
    <addaction name="viewCustomSizeAction"/>
   </widget>
   <widget class="QMenu" name="playMenu">
    <property name="title">
     <string>&amp;Play</string>
    </property>
    <widget class="QMenu" name="audiotrackpopup">
     <property name="title">
      <string>Audio track</string>
     </property>
    </widget>
    <addaction name="playPlayAction"/>
    <addaction name="playStopAction"/>
    <addaction name="separator"/>
    <addaction name="playAudio1Action"/>
    <addaction name="playAudio2Action"/>
    <addaction name="separator"/>
    <addaction name="audiotrackpopup"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction"/>
    <addaction name="helpContentAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="playMenu"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileOpenAction"/>
   <addaction name="fileSaveAction"/>
   <addaction name="fileSaveAsAction"/>
   <addaction name="snapshotSaveAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="editStartAction"/>
   <addaction name="editStopAction"/>
   <addaction name="editChapterAction"/>
   <addaction name="editBookmarkAction"/>
  </widget>
  <widget class="QToolBar" name="playToolbar">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="playAudio1Action"/>
   <addaction name="playAudio2Action"/>
   <addaction name="playPlayAction"/>
   <addaction name="playStopAction"/>
  </widget>
  <action name="fileNewAction">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="fileSaveAsAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="snapshotSaveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save Snapshot</string>
   </property>
  </action>
  <action name="chapterSnapshotsSaveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Chapter Snapshots</string>
   </property>
  </action>
  <action name="fileExportAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export video...</string>
   </property>
  </action>
  <action name="fileCloseAction">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="editStartAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dvbcutbase.qrc">
     <normaloff>:/dcb/icons/play.png</normaloff>:/dcb/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Set start marker</string>
   </property>
  </action>
  <action name="editStopAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dvbcutbase.qrc">
     <normaloff>:/dcb/icons/stop.png</normaloff>:/dcb/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Set stop marker</string>
   </property>
  </action>
  <action name="editChapterAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dvbcutbase.qrc">
     <normaloff>:/dcb/icons/chapter.png</normaloff>:/dcb/icons/chapter.png</iconset>
   </property>
   <property name="text">
    <string>Set chapter marker</string>
   </property>
  </action>
  <action name="editBookmarkAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dvbcutbase.qrc">
     <normaloff>:/dcb/icons/bookmark.png</normaloff>:/dcb/icons/bookmark.png</iconset>
   </property>
   <property name="text">
    <string>Set bookmark</string>
   </property>
  </action>
  <action name="editAutoChaptersAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto chapters</string>
   </property>
  </action>
  <action name="editSuggestAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Suggest bookmarks</string>
   </property>
  </action>
  <action name="editImportAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import bookmarks</string>
   </property>
  </action>
  <action name="viewNormalAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="viewUnscaledAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unscaled</string>
   </property>
  </action>
  <action name="viewDifferenceAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show difference to current picture</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="viewFullSizeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full size</string>
   </property>
  </action>
  <action name="viewHalfSizeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Half size</string>
   </property>
  </action>
  <action name="viewQuarterSizeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quarter size</string>
   </property>
  </action>
  <action name="viewCustomSizeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom size</string>
   </property>
  </action>
  <action name="playPlayAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="playStopAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="playAudio1Action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-skip-forward">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Play audio: last 2 seconds</string>
   </property>
   <property name="iconText">
    <string>Audio -&gt;|</string>
   </property>
  </action>
  <action name="playAudio2Action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-skip-backward">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Play audio: next 2 seconds</string>
   </property>
   <property name="iconText">
    <string>Audio |-&gt;</string>
   </property>
  </action>
  <action name="helpAboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="helpContentAction">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
  </action>
  <action name="actionFoo">
   <property name="text">
    <string>foo</string>
   </property>
  </action>
  <action name="actionFoo_2">
   <property name="text">
    <string>foo</string>
   </property>
  </action>
  <action name="actionFoo_3">
   <property name="text">
    <string>foo</string>
   </property>
  </action>
  <action name="editConvertStart">
   <property name="text">
    <string>START / STOP</string>
   </property>
  </action>
  <action name="editConvertStop">
   <property name="text">
    <string>STOP / START</string>
   </property>
  </action>
  <action name="editConvert4_3">
   <property name="text">
    <string>4 : 3</string>
   </property>
  </action>
  <action name="editConvert16_9">
   <property name="text">
    <string>16 : 9</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="dvbcutbase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileCloseAction</sender>
   <signal>triggered()</signal>
   <receiver>dvbcutbase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goinput</sender>
   <signal>returnPressed()</signal>
   <receiver>gobutton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goinput2</sender>
   <signal>returnPressed()</signal>
   <receiver>gobutton2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
